parts/Compound/CompoundDialogs/index.js

```diff
@@ -100,10 +100,11 @@ const CompoundDialogs = ({
     error: null
   })
 
-  const { approve, deposit, isTransacting, transactionStatus, withdraw } = useCompoundAndEarnContract();
+  const { approve, deposit, isTransacting, transactionStatus } = useCompoundAndEarnContract();
+  const { AVAXBalance } = useContracts();
 
   useEffect(() => {
-    if (!isTransacting.deposit && !isTransacting.approve && !isTransacting.withdraw) {
+    if (!isTransacting.deposit && !isTransacting.approve) {
```

Comment:
why are we removing the ability to withdraw again? to be clear this is more of a biz question than a code question

Reply:
This code was redundant, when this PR started we used the same component for withdrawing and depositing, now we have it two separated since they will be very distinct from now on.